yt/__init__.py

```diff
@@ -176,3 +176,16 @@
 from yt.units.unit_systems import UnitSystem, unit_system_registry
 
 _called_from_pytest = False
+
+
+def _check_deprecated_parameters():
+    from yt.config import ytcfg
+    from yt.funcs import issue_deprecation_warning
+
+    if ytcfg.getboolean("yt", "loadfieldplugins"):
```

Comment:
If this load field plugins was added since the last release I don't think we need to issue a deprecation warning.

Reply:
Indeed it seems to be the opposite: this is an old option that apparently fell out of use but we have a few checks still around for backward compatibility
See this comment
https://github.com/yt-project/yt/blob/7f488b615a7cf42bdabb46eeb4eb7a259665b4e4/yt/utilities/command_line.py#L48

Reply:
ahhh ok I thought it had been more recently added!